Bottom: 762ddecbef8d1c5e22a9625dc464886ed3f7c1c5
Top:    99ddaea865f548cfe3fa2bf3086138ac132f896d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 19:21:06 +0100

Refresh of change-penalty-function-in-rtr

---

diff --git a/src/rtree.c b/src/rtree.c
index 59793f8..3b30a56 100644
--- a/src/rtree.c
+++ b/src/rtree.c
@@ -79,7 +79,6 @@ RCSID ("$Id$");
 #define SORT_NONLEAF
 
 #define DELETE_BY_POINTER
-typedef long long bigun;
 
 typedef struct
 {
